pkg/nvpci/nvpci_test.go

```diff
@@ -74,6 +74,33 @@ func TestNvpci(t *testing.T) {
 	_, err = nvpci.GetGPUByIndex(1)
 	require.Error(t, err, "No error returned when getting GPU at invalid index")
 }
+func TestNvpciIOMMUFD(t *testing.T) {
```

Comment:
Seems like we need a newline before this?
```suggestion

func TestNvpciIOMMUFD(t *testing.T) {
```